applications/GeoMechanicsApplication/custom_retention/van_genuchten_law.cpp

```diff
@@ -29,17 +26,17 @@ double VanGenuchtenLaw::CalculateSaturation(Parameters& rParameters) const
 {
     KRATOS_TRY
 
-    const double&     p                   = rParameters.GetFluidPressure();
-    const Properties& rMaterialProperties = rParameters.GetMaterialProperties();
+    const auto  p                   = rParameters.GetFluidPressure();
+    const auto& rMaterialProperties = rParameters.GetMaterialProperties();
```

Comment:
To be in line with the Kratos Style Guide:
```suggestion
    const auto& r_material_properties = rParameters.GetMaterialProperties();
```

Reply:
done trhoughout.